microscope/filterwheels/aurox.py

```diff
@@ -157,11 +166,14 @@ def _c_process_data(self, data):
             raise Exception("Unrecognised mode: %s", self._mode)
 
     @property
-    def devices(self) -> typing.Mapping[str, microscope.devices.Device]:
-        return self._devices
+    def devices(self) -> Mapping[str, microscope.devices.Device]:
+        """Devices property, required by ControllerDevice interface."""
+        return {'camera': self._cam}
```

Comment:
If there is no `CameraDevice` this should return an empty dict, and not a camera with a `None` value. Also, the possible devices of a controller should be documented.

Reply:
Returning ```{}``` indicates "I don't have any devices", but ```{'camera': None}``` indicates I should have a camera but don't". Is this more useful, or maybe a different use case? I guess this applies more to the Clarity because it's not much use without a camera, whereas other controllers may aggregate a bunch of optional devices (e.g. a stage controller that may have a variable number of axes attached).

Reply:
Where should the documentation go?

Reply:
We are specifically trying to avoid having `None`s as meaning that it could have that device but does not. See issue #111 but basically it's tricky to derive what the full list of possible devices is. Examples: having a device connected may prevent others from being connected (the filterwheel 3 on prior proscan prevents having a light source; the ZaberDaisyChain controller can have up to 255 devices connected.

On the documentation about what controlled devices may exist, it should go on the docstring of the class that implements the Controller interface.

Reply:
c5e5270: Clarity now returns {} if no camera instead of {'camera': None}

Reply:
Docstrings added in 0c98ba9.